internal/grpc/services/storageprovider/storageprovider.go

```diff
@@ -65,6 +66,7 @@ type config struct {
 	AvailableXS         map[string]uint32                 `mapstructure:"available_checksums" docs:"nil;List of available checksums."`
 	CustomMimeTypesJSON string                            `mapstructure:"custom_mimetypes_json" docs:"nil;An optional mapping file with the list of supported custom file extensions and corresponding mime types."`
 	MountID             string                            `mapstructure:"mount_id"`
+	UploadExpiration    int64                             `mapstructure:"upload_expiration" docs:"0;Duration for how long uploads will be valid."`
```

Comment:
Do we need a new config value? Didn't we just want to use the `TransferExpires` value?

Reply:
The default values are identical and they probably won't be changed by many people but since the setting **is** specific to the storage I would rather we make it possible to actually configure it.